lib/minidoc/counters.rb

```diff
@@ -27,10 +27,10 @@ def initialize(record, field)
       end
 
       def increment(step_size = 1)
-        result = record.class.collection.find_and_modify(
-          query: { _id: record.id },
-          update: { "$inc" => { field => step_size } },
-          new: true,
+        result = record.class.collection.find_one_and_update(
+          { _id: record.id },
+          { "$inc" => { field => step_size } },
+          return_document: :after,
```

Comment:
`new: true` is documented here: https://github.com/mongodb/mongo-ruby-driver/blob/1.8.4/lib/mongo/collection.rb#L608

It specifies which document the method should return, the version before the update (`new: false`, the default) or the version after the update (`new: true`)

I think `return_document: after` is the same idea with a different name. It's documented here: https://github.com/mongodb/mongo-ruby-driver/blob/v2.4.1/lib/mongo/collection.rb#L572